File: Belajar-loop2/loop2.py
'''
 Level1

 1
 23
 456
 78910

 Level 5

    *
   ***
  *****
 *******

 '''

a = 4
b = 1
for i in range(a):
    for j in range(i+1):
        print(b, end="")
        b+=1
    print()
print()
for i in range(0,a*2,2):
    for j in range(a-i//2+1):
        print(" ", end="")
    for j in range(i+1):
        print("*",end="")
    print()

File: belajar-loop/Belajar-loop2/loop2.rs

/*
 * Level1

 1
 23
 456                                        
 78910

 *Level 5

    *
   ***
  *****
 *******

 */
fn main() {
    let a: u8 = 4;
    let mut b: u8 = 1;
    for i in 0..a {
        for _j in 0..=i {
            print!("{}",b);
            b+=1;
        }
        println!();
    }
    println!();
    for i in (0..a*2).step_by(2) {
        for _j in 0..a-i/2 {
            print!(" ");
        }
        for _j in 0..=i {
            print!("*");
        }
        println!();
    }
}
